diff --git a/node_modules/react-native-audio-player/android/build.gradle b/node_modules/react-native-audio-player/android/build.gradle
index f7a1e41..fb3bc29 100644
--- a/node_modules/react-native-audio-player/android/build.gradle
+++ b/node_modules/react-native-audio-player/android/build.gradle
@@ -16,6 +16,6 @@ android {
 }
 
 dependencies {
-  compile 'com.facebook.react:react-native:0.11.+'
+  implementation 'com.facebook.react:react-native:0.11.+'
 }
 
